cni/network/plugin/main.go

```diff
@@ -295,14 +296,21 @@ func main() {
 		os.Exit(0)
 	}
 
-	loggerCfg := &log.Config{
+	log.SetName(name)
+	log.SetLevel(log.LevelInfo)
+	if err := log.SetTargetLogDirectory(log.TargetLogfile, ""); err != nil {
+		fmt.Printf("Failed to setup cni logging: %v\n", err)
+		return
+	}
+
```

Comment:
can we move this before `NewPlugin` api is called?

Reply:
Hi Tamilmani, can you please explain more about this? This is already before NewPluigin(rootExecute()) is called.

Reply:
ok then its fine